README.md

```diff
@@ -20,29 +20,31 @@ The project is adapted from the [`ros_gz_project_template`](https://github.com/g
 ## Prerequisites
 
 - Install [ROS 2 Humble](https://docs.ros.org/en/humble/index.html)
-- Install [Gazebo Garden](https://gazebosim.org/docs/garden)
+- Install [Gazebo Harmonic (recommended)](https://gazebosim.org/docs/harmonic) or [Gazebo Garden](https://gazebosim.org/docs/garden)
 - Follow the [`Installing Build Dependencies`](https://github.com/ArduPilot/ardupilot/tree/master/libraries/AP_DDS#installing-build-dependencies) section of `AP_DDS`'s README
 
 ## Install
 
 #### 1. Create a workspace folder
 
 ```bash
-mkdir -p ~/ros2_ws/src && cd ~/ros2_ws/src
+mkdir -p ~/ros2_ws/src && cd ~/ros2_ws
 ```
 
 #### 2. Get the project source
 
 ```bash
-cd ~/ros2_ws/src
-wget https://raw.githubusercontent.com/ArduPilot/ardupilot_gz/main/ros2_gz.repos
-vcs import --recursive < ros2_gz.repos
+cd ~/ros2_ws
+mkdir src
```

Comment:
Don't need this as created above in L31.

Reply:
Good catch. a0ebc9eca53d011df25f034b6c0f23cff02672c4